packages/shader-lab/src/parser/AST.ts

```diff
@@ -219,18 +219,20 @@ export namespace ASTNode {
 
   @ASTNodeDecorator(NoneTerminal.fully_specified_type)
   export class FullySpecifiedType extends TreeNode {
-    get qualifierList() {
-      if (this.children.length > 1) {
-        return (<TypeQualifier>this.children[0]).qualifierList;
-      }
-    }
-
-    get typeSpecifier() {
-      return (this.children.length === 1 ? this.children[0] : this.children[1]) as TypeSpecifier;
-    }
+    qualifierList: EKeyword[] = [];
```

Comment:
Why can't I find where `qualifierList` is used? Do we need to calculate this property now?